cmd/bad-key-revoker/main.go

```diff
@@ -128,9 +128,6 @@ func (bkr *badKeyRevoker) findUnrevoked(unchecked uncheckedBlockedKey) ([]unrevo
 				string(core.StatusRevoked),
 			)
 			if err != nil {
-				if db.IsNoRows(err) {
```

Comment:
This has the same problem as before, there are legitimate cases where the we don't expect any rows to come back, i.e. the integration test cases.

If we revoke a certificate with admin-revoker the key hash is added to the blocked key list, we then check that list in bad-key-revoker and get the hash, we lookup any certificates matching the key hash and get the serial of the certificate that was already revoked, we lookup that certificate status, but we don't get anything because it's status is already set to revoked.

Reply:
Ah, good point, will rework. Thanks!